pkg/plaid_cli/linker.go

```diff
@@ -179,19 +207,19 @@ var linkTemplate string = `<html>
   <head>
     <style>
     .alert-success {
-        font-size: 1.2em;
```

Comment:
😑 I guess Emacs decided to make some whitespace changes. Happy to revert these if that's not welcome.